### DISCUSSION TITLE
Seedlings and Beyond?

### DISCUSSION THREAD

**Discussion by Dalnorae** [BryanJenksCommunity/FAQ#76] [Q&A] [answered]
Hello Bryan!

A quick question. I saw that you had three stages for evergreen notes. When would you label a note as a seedling (earliest)? I ask because from what I’ve seen you didn’t label EVERYTHING as a seedling. Just curious about your organizing methods. I find myself the urge to have everything within those three stages and it ultimately feels less productive.

**Comment by tallguyjenks** [comment, marked as answer]
I have 4 now! the order of progression is:

🌱️ --> 🌿️ --> 🌞️ --> 🌲️

- 🌱️ Could be an empty note or a brand new idea, just needs a LOT of work
- 🌿️ Could have been a half baked idea, or something I've expanded on a bit more than a 🌱️
- 🌞️ this is where I might have finished with a 🌿️ but I might now have a good place to connect it or Im working on a batch of ideas and I want to look at a pool of them at once and hold them here
- 🌲️Evergreen notes :)

Relevant & Related:

- [[[Tag Taxonomy]]](https://publish.obsidian.md/bryan-jenks/Tag+Taxonomy)
- #71 
- #49

**Reply by Dalnorae** [reply]
Thank you for the clear explanation. 

And I wanted to get your feedback on something and what it would look like farther down the road when it comes to organization. I only have around 150 notes at this point and not sure if it would clear up or mess up when I get more notes. Not much a question so I think it'll be better to stick it in here haha.

SO, I'm currently using your method of organizing tags, and I see how useful it is. The nodes get bigger and I know that's something I can develop into an MOC. But would it be worthwhile to organize the "tags" when we make them into an MOC?

For example, two huge nodes right now are "productivity" and "obsidian." MOST of the notes with the "obsidian" tag also have "productivity" tag. And "productivity" is apparently higher in the hierarchy. SO, I'm thinking of making "obsidian" into an MOC/TagNote and deleting the tag "productivity" from all notes that are organized within it. This way, it clears up the node view and makes it easier to see which nodes need to be developed.

Basically, I would leave any in-text backlinks, while deleting all but one proper link that will ultimately lead to HOME MOC. I'm hesitant because I feel like I may be sacrificing the "web" aspect of Obsidian, but at the same time, I feel like leaving the in-text backlinks while giving it a clearer hierarchical structure through MOC could be more efficient.

Of course, I would only delete "tags" if they are related on a hierarchical level. So if I had three tags: "productivity" | "obsidian" | "philosophy" I would delete "productivity" if I made an "obsidian" MOC/TagNote which falls under "productivity"
